lib/models/device.coffee

```diff
@@ -509,6 +544,76 @@ exports.move = (uuid, application, callback) ->
 	.nodeify(callback)
 
 ###*
+# @summary Start application on device
+# @name start
+# @public
+# @function
+# @memberof resin.models.device
+#
+# @param {String} uuid - device uuid
+# @fulfil {String} - application container id
```

Comment:
The function resolves the response body, which is an object containing a `containerId` property, but not the property itself. You can add `.get('containerId')` after `.get('body')` so the functionality works according to this claim.